lib/debts/letter_downloader.rb

```diff
@@ -29,10 +29,13 @@ def get_letter(document_id)
       ).content
     end
 
+    def file_name(document_id)
```

Comment:
is it possible to add a test for this new method?

Reply:
Done